Enterprise-CMCS/macpro-onemac
# Emails: Sub Doc Email Fix

Story: https://qmacbis.atlassian.net/browse/OY2-30491 &
           https://qmacbis.atlassian.net/browse/OY2-30492
           
Email Figma Referencing: https://www.figma.com/design/d5L1JdWrZM3mRClaufixV6/Subsequent-Submissions---Content---OY2-28538?node-id=28-49&node-type=canvas&t=EpjRvHxvSvNOS0R4-0
Endpoint: See github-actions bot comment

### Details

Minor fixes to Sub Doc Emails

### Changes

- Made a copy object for CMS sub sub emails so that we aren't actually changing anything in the data object
- Also removed bold on the file heading to match confluence/figma

### Screenshots
CMS Email 
![Screenshot 2024-09-30 at 3 13 40 PM](https://github.com/user-attachments/assets/9f080d34-45a4-4762-849b-4c5e03d4f2b3)

State Email
![Screenshot 2024-09-30 at 3 13 58 PM](https://github.com/user-attachments/assets/4c78de30-2beb-4ad3-83ca-816f1e877077)

### Test Plan

1. Login as state user
2. Find package that allows subsub
3. Submit sub docs 
4. Check emails match the figma
